src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -89,9 +97,12 @@
                 {{ product.quantity }}
               {% endif %}
             </td>
+            <td>
+              <p>{{ product.quantityRefunded }}{% if product.amountRefund() %} ({{ product.amountRefund() }} {{ 'Refunded'|trans({}, 'Admin.Orderscustomers.Feature') }}){% endif %}</p>
```

Comment:
Is the `()` necessary? It it's an `OrderProductForViewing` object then you can simply use `product.amountRefund`